dotCMS/src/main/java/com/dotcms/system/event/local/type/publish/PublishEvent.java

```diff
@@ -0,0 +1,43 @@
+package com.dotcms.system.event.local.type.publish;
+
+import com.dotcms.publisher.business.PublishQueueElement;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Object used to represent an event to be triggered among a push publishing process, ie: assets
+ * added to queue, push publishing starts, push publishing ends, etc
+ *
+ * @author nollymar
+ */
+public class PublishEvent {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)